src/types.ts

```diff
@@ -90,6 +96,7 @@ export type CSPSourceValue =
   | "'none'"
   | "'strict-dynamic'"
   | "'report-sample'"
+  | "'nonce=<base64-value>'"
```

Comment:
Is it corrected for type? May be this: `'nonce=${string}'`

Reply:
It's generally suggested to use a base64 encoded random string.
The current implementation does that, but it's not a hard requirement per se.